docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3070,7 +3076,9 @@ would be listed in MM/DD/YYYY format as 08/12/2022).
 
 Incorrect select applies to a range of data types. For this, select the sample to 
 have noise added. For those selected, randomly select a different option from the 
-others available. Note that for relationship to head of household, this includes 
+correct one. This is randomly chosen from the original list of options 
+(e.g., for state, if WA is the true answer randomly pick any other state). 
```

Comment:
The issue is that even though this "list of options" makes perfect intuitive sense, it doesn't really exist anywhere.

For example, when we initialize simulants to be in a state, we don't choose from a list of states; instead, we sample a row from the 15M rows in the ACS PUMS to base that simulant on, and that row already has a state attribute.